frameworks/CSharp/aspnet/benchmark_config.json

```diff
@@ -203,7 +203,7 @@
       "notes": "",
       "versus": ""
     },
-    "mono": {
+    "default": {
```

Comment:
I tried running aspnet, not aspnet-mono, so that might be the issue. Still even that should not fail (in my opinion)